contracts/LockedNORI.sol

```diff
@@ -3,50 +3,101 @@ pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts-upgradeable/utils/math/MathUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/token/ERC777/ERC777Upgradeable.sol";
-import "@openzeppelin/contracts-upgradeable/token/ERC20/presets/ERC20PresetMinterPauserUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/token/ERC777/IERC777RecipientUpgradeable.sol";
 import "./NORI.sol";
 import {ScheduleUtils, Schedule, Cliff} from "./ScheduleUtils.sol";
 
 /**
- * @title {ERC20Wrapper} extension implementing scheduled {NORI} token vesting + lockup.
+ * @title A wrapped NORI token contract for vesting and lockup
  * @author Nori Inc.
+ * @notice Based on the mechanics of a wrapped ERC-777 token, this contract layers in schedules to withdrawal
```

Comment:
This description got a bit scrambled but I think we should merge this and then I'll give it another pass.

Reply:
(before this PR)